docs/integrator/getting-started.md

```diff
@@ -18,7 +18,7 @@ With this basic structure in place, you're ready to develop an ad plugin.
 
 ### Including the files in your HTML
 
-This is not the recommended approach, but may be useful in some cases. In addition to the video.js library, you'll need two files from this project: `videojs.ads.js` and `videojs.ads.css`. After you [build the project](/developer/getting-started.md) they are both in the `dist` directory.
+This is not the recommended approach, but may be useful in some cases. In addition to the video.js library, you'll need two files from this project: `videojs.ads.js` and `videojs.ads.css`. After you [build the project](/docs/developer/getting-started.md) they are both in the `dist` directory.
```

Comment:
given that these docs are supposed to be read on [gh-pages](https://videojs.github.io/videojs-contrib-ads/integrator/getting-started.html), I think this should actually be `../developer/getting-started.md`
```suggestion
This is not the recommended approach, but may be useful in some cases. In addition to the video.js library, you'll need two files from this project: `videojs.ads.js` and `videojs.ads.css`. After you [build the project](../developer/getting-started.md) they are both in the `dist` directory.
```